src/Tests/dotnet-workload-install.Tests/MockPackWorkloadInstaller.cs

```diff
@@ -7,6 +7,7 @@
 using static Microsoft.NET.Sdk.WorkloadManifestReader.WorkloadResolver;
 using Microsoft.DotNet.Workloads.Workload.Install.InstallRecord;
 using Microsoft.Extensions.EnvironmentAbstractions;
+using System.Linq;
 
 namespace Microsoft.DotNet.Cli.Workload.Install.Tests
```

Comment:
We should test and handle (better error message) when older sdk cannot read higher version manifest file

Reply:
Can you clarify the scenario you're talking about? I don't think we need to worry about an older SDK reading a newer manifest. The way it's implemented right now, we only ever look at the current feature band's manifests. The steps are basically figure out which workloads are installed with the prior feature band (using workload install records) and then determine which packs we need for those workloads based on the current band's manifests.

Reply:
Just pushed test coverage/ better checks for valid sdk-version option values. Is this along the lines of what you were thinking?